Website/index.js

```diff
@@ -0,0 +1,472 @@
+function show_personalised() {
+    $("#updated").text(jsonData["updated"]);
+    $("#date2").remove();
+
+    $("#personalised").empty();
+
+    function DaysOfWeek(day) {
+        $(Object.keys(jsonData["weeks"])).each(function (i, v) {
+
+            $(jsonData["weeks"][v]).each(function (b, value) {
+                var a = new Date(value);
+
+                if (String(day) == String(a)) {
+                    week = jsonData["weeks"][v]
+                }
+
+            });
+
+        });
+    }
+
+    var day = new Date(String($("#start").val()));
+    DaysOfWeek(day);
+    var week;
+
+    var per_dict = {
+        0: {
+            "id": "day_1",
+            "h4": "Monday",
+        },
+        1: {
+            "id": "day_2",
+            "h4": "Tuesday",
+        },
+        2: {
+            "id": "day_3",
+            "h4": "Wednesday",
+        },
+        3: {
+            "id": "day_4",
+            "h4": "Thursday",
+        },
+        4: {
+            "id": "day_5",
+            "h4": "Friday"
+        }
+    }
+
+    var recess_badge = {
+        1: '<span class="badge bg-primary mb-1">First Recess (9:10 - 9:45)</span>',
+        2: '<span class="badge bg-secondary mb-1">Second Recess (9:45 - 10:20)</span>',
+        3: '<span class="badge bg-success mb-1">Third Recess (10:20 - 10:55)</span>',
+        4: '<span class="badge bg-info mb-1">Fourth Recess (10:55 - 11:30)</span>'
+    }
+
+    var duty_trans = {
+        "nb1": "NB1",
+        "nb2": "NB2",
+        "nb3": "NB3",
+        "stall": "Stall",
+        "pond": "Pond",
+        "field": "Field",
+        "parade_square": "Parade Square",
+        "lift": "Lift",
+        "central": "Central",
+        "patrol": "Patrol",
+        "on_break": "On Break",
+        "main_gate": "Main Gate",
+        "late_coming": "Late Coming",
+        "canteen": "Canteen",
+        "stairs_to_hall": "Stairs To Hall",
+        "side_gate": "Side Gate",
+        "morning_patrol": "Morning Patrol",
+        "hall": "Hall"
+    }
+
+    var counter = 0;
+    var days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Davianlols6/SC-Duty-List/pullRequest?prid=8386818)
```suggestion
    var days = ["Sunday", 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
```